### PR TITLE
OSTestInFailure exception added.

### PR DESCRIPTION
**Pull request by rhopp**

### CODE REVIEW

File: tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/create/CreateApplicationOnBuilderImageTest.java [hunk 1]
```diff
@@ -127,7 +128,11 @@ private void validateJBIDE22704() {
 		String gitUrl = new LabeledText("Git Repository URL:").getText();
 
 		assertNotNull(gitUrl);
-		assertTrue("Git URL is empty (JBIDE-23704)", gitUrl.length() > 0);
+		if (gitUrl.length() < 1){
+			new CancelButton().click();
```
Comment (rawagner) [inline tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/create/CreateApplicationOnBuilderImageTest.java]: I would suggest to add wait for shell that is supposed to be cancelled

Comment (rhopp) [inline tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/create/CreateApplicationOnBuilderImageTest.java]: This is already done here: https://github.com/jbosstools/jbosstools-openshift/blob/master/test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/wizard/AbstractOpenShiftApplicationWizard.java
On line 71 (case of validateJBIDE22704FromShellMenu) or on line 115 (case of validateJBIDE22704FromCentral)

Comment (rhopp) [inline tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/create/CreateApplicationOnBuilderImageTest.java]: In other words this whole method expects the shell to be already present.

**Comment by rhopp** [comment]
@rawagner I've added the wait for the shell. Right now it's just WaitWhile(new ShellWithTextIsAvailable). Possible rewrite to ShellIsAvailable would need bigger refactoring.

**Pull request merged**